production/scrapers/oklahoma_manual.R

```diff
@@ -1,6 +1,19 @@
 source("./R/generic_scraper.R")
 source("./R/utilities.R")
 
+oklahoma_vaccine_check_date <- function(x, date = Sys.Date()){
```

Comment:
Might want to call this `oklahoma_manual_check_date` (sadly no vaccine data from OK).